### PR TITLE
Fix: Abstract models should not show 'database table' string in rendered documentation

### LINKED ISSUE #39: Abstract models should not show 'database table' string in rendered documentation
In cases when documentation generated for module with abstract models, currently 'Database table:' is rendered. This is incorrect, as Abstract models does not have any database representation.

### PR DESCRIPTION
**Pull request by insspb**
This fixes issue, raised by me in #39 

It also rename database model related options, and add few other.

fixes #39

### CODE REVIEW

**Comment by insspb** [comment]
@timoludwig, please take a look. I am plan to add few more pull request on things, that bothers me.

But as all of them related to model generation, I do not want to add all at once, to exclude merge conflicts.

File: README.rst [hunk 1]
```diff
@@ -75,12 +75,17 @@ Add the following to your Sphinx config file ``conf.py``:
...
 .. code-block:: python
 
     # Include the database table names of Django models
-    django_show_db_tables = True
+    django_tables_names_show = True                # Boolean, default: False
+    # Add or ignore abstract database tables names, as
+    # `django_tables_names_abstract_name`, to keep same style formatting
+    django_tables_names_ignore_abstract = False    # Boolean, default: True
+    # Abstract models name placeholder replacement, default: None
+    django_tables_names_abstract_name = None       # String or None
```
[the remark this answers is not in the document]
Comment (insspb) [reply README.rst:88]: @timoludwig 
Ok. Let's do `None` in cases, when it is abstract table and we have global tables showing. 
And no string, if we use `django_tables_names_ignore_abstract` = `True` (default)

**Comment by insspb** [comment]
@timoludwig I removed django_tables_names_abstract_name option. 
Second PR (#42)  fixed too.

**Review by timoludwig — APPROVED** [review]
@insspb Thanks a lot for your changes and sorry for the delay!

I reverted your renaming of the config variable to allow backward compatibility and added a changelog entry, but other than that it looks good! :+1:

**Pull request merged**